t/t5324-split-commit-graph.sh

```diff
@@ -4,6 +4,7 @@ test_description='split commit graph'
 . ./test-lib.sh
 
 GIT_TEST_COMMIT_GRAPH=0
+GIT_TEST_COMMIT_GRAPH_BLOOM_FILTERS=0
```

Comment:
you need both of these lines in `t5325-commit-graph-bloom.sh`, too.

```
GIT_TEST_COMMIT_GRAPH=0
GIT_TEST_COMMIT_GRAPH_BLOOM_FILTERS=0
```
Currently, this fails:
```
GIT_TEST_COMMIT_GRAPH=1 prove -j8 t5325-commit-graph-bloom.sh 
```